packages/web/src/components/gcds-side-nav/stories/gcds-side-nav.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { langProp } from '../../../utils/storybook/component-properties';
+
+export default {
+  title: 'Components/Side navigation',
+
+  argTypes: {
+    // Props
+    label: {
+      name: 'label',
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' }
+      },
+      type: {
+        required: true
+      }
+    },
+    // position: {
+    //   control: { type: 'radio' },
+    //   options: ['static', 'sticky'],
+    //   table: {
+    //     type: { summary: 'string' },
+    //     defaultValue: { summary: 'static' }
+    //   },
+    // },
+    ...langProp,
+  },
+};
+
+const Template = (args) => (`
+// Web Component (Angular, Vue)
```

Comment:
Can we update this comment to `<!-- Web component code (Angular, Vue) -->` so that it doesn't show in the preview?